engine/lib/phx/script/ffi/Sound.lua

```diff
@@ -5,81 +5,39 @@ local Sound
 
 do -- C Definitions
     ffi.cdef [[
-    void   Sound_Acquire               (Sound*);
-    void   Sound_Free                  (Sound*);
-    Sound* Sound_Load                  (cstr name, bool isLooped, bool is3D);
-    Sound* Sound_LoadAsync             (cstr name, bool isLooped, bool is3D);
-    Sound* Sound_Clone                 (Sound*);
-    void   Sound_ToFile                (Sound*, cstr);
-    void   Sound_Pause                 (Sound*);
-    void   Sound_Play                  (Sound*);
-    void   Sound_Rewind                (Sound*);
-    bool   Sound_Get3D                 (Sound*);
-    float  Sound_GetDuration           (Sound*);
-    bool   Sound_GetLooped             (Sound*);
-    cstr   Sound_GetName               (Sound*);
-    cstr   Sound_GetPath               (Sound*);
-    bool   Sound_IsFinished            (Sound*);
-    bool   Sound_IsPlaying             (Sound*);
-    bool   Sound_IsAudible             (Sound*);
-    void   Sound_Attach3DPos           (Sound*, Vec3f const* pos, Vec3f const* vel);
```

Comment:
We removed quite a lot of functions here, are we certain we don't need them?

Reply:
We don't use them, that's for sure. We can add them later if needed.